pkg/gopass/secrets/yaml.go

```diff
@@ -41,14 +41,14 @@ func (y *YAML) Keys() []string {
 	for key := range y.data {
 		keys = append(keys, key)
 	}
+	sort.Strings(keys)
 	if _, found := y.data["password"]; !found {
-		keys = append(keys, "password")
+		keys = append([]string{"password"}, keys...)
```

Comment:
You seem to always order the key `password` first, which is good.
But maybe we should write down that assumption somewhere?

Reply:
Yeah, since we said that the secret would always be on the first line, I figured that we might as well always have the password key first (unless there's a password key in the secret, but maybe we might want to change that to have multiple password keys in that case?)
We might also want to include a check that the password is non-null, no?

Reply:
Hmm, thinking more about it this is a bit weird.
The `key` `password` isn't necessarily the same as the `password` entry (the first line).

So we could very well have a password field (first line) and one - or even multiple - (KV or YAML) keys named `password`.

Reply:
Yes, exactly. Notice that if the key `password` is in the entry, the extra `password` key at the beginning is skipped, the rational was that currently if we have multiple "same" key, we do not display them in the same order as they appear in the secret since we sort them first and so a `password` key in the body should be sorted with the other keys...
 
We could keep track of the key order and display multiple keys if we were to add a field to the KV type to retain the keys and their order upon parsing... WDYT?

Reply:
Uh, that sounds like a lot of extra complexity. I'd rather not do that.

Reply:
Thinking more about this: please drop the if block (line 45-47).
We don't need it and it doesn't make much sense, i.e. you won't get the content of the first line with a `Get("password")` anyway. And if there is one (or multiple) key with the name `password` we can handle it like any other key.